css/templates/e-commerce/page.css

```diff
@@ -701,6 +747,9 @@ amp-selector [option][selected] {
   border-bottom: 1px dashed #dadada;
 }
 
+.amp-mode-mouse .commerce-listing-product:hover > .commerce-listing-product-name {
```

Comment:
Some mobile devices still trigger the hover state on tap and remain until the user taps or navigates elsewhere. For this reason we only want to target devices that support a mouse pointer.

See first "Note" in docs for :hover
https://developer.mozilla.org/en-US/docs/Web/CSS/:hover